app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module CommentsHelper
+  def comment_formatter(comment)
+    @comment_pipeline ||= HTML::Pipeline.new [
+      HTML::Pipeline::DradisCommentFilter,
+      HTML::Pipeline::DradisMentionsFilter,
+    ], {
+      mentionable_users: @mentionable_users || current_project.authors,
```

Comment:
we already did `|| current_project.authors` in `find_mentionable_users` so we can skip it here?

Reply:
See this comment: https://github.com/dradis/dradis-ce/pull/548#discussion_r348703802